rules/src/main/scala/fix/InterpolationToString.scala

```diff
@@ -16,9 +16,13 @@ class InterpolationToString extends SyntacticRule("InterpolationToString") {
             values
           ) =>
         values.collect {
-          case x1 @ Term.Select(
-                x2,
-                Term.Name("toString")
+          case Term.Block(
```

Comment:
will be included in the release notes as a breaking change, see https://github.com/scalameta/scalameta/issues/3594